test/PrefilterExpressionTestHelpers.h

```diff
@@ -97,6 +106,21 @@ auto makePrefilterVec =
 
 }  // namespace makeFilterExpression
 
+// _____________________________________________________________________________
+// Helper to retrieve a `ValueId` from `IdOrLocalVocabEntry`.
+const auto getValueIdFromIdOrLocalVocabEntry =
+    [](const prefilterExpressions::IdOrLocalVocabEntry& value,
+       LocalVocab& localVocab) {
+      return std::visit(
+          ad_utility::OverloadCallOperator{
+              [](const ValueId& referenceId) { return referenceId; },
+              [&localVocab](const LocalVocabEntry& referenceValue) {
+                return Id::makeFromLocalVocabIndex(
+                    localVocab.getIndexAndAddIfNotContained(referenceValue));
+              }},
+          value);
+    };
+
```

Comment:
If I am not mistaken, this is an exact duplicate from a lambda inside the `PrefilterExpressionClass`.
So make it a (static) member function there, and just reuse it in the tests without duplicating it.